examples/game_mode.py

```diff
@@ -0,0 +1,53 @@
+import time
+
+from agents.random_agent import RandomAgent
+from gym_xiangqi.constants import (     # NOQA
+    RED, BLACK, PIECE_ID_TO_NAME, AGENT
+)
+from gym_xiangqi.utils import action_space_to_move
+from gym_xiangqi.envs import XiangQiEnv
+
+
+def main():
+    # Pass in the color you want to play as (RED or BLACK)
+    env = XiangQiEnv(RED)
+    env.render()
+    agent = RandomAgent()
+
+    done = False
+    round = 0
+    while not done:
+        if env.turn == AGENT:
```

Comment:
Oh this seems a bit counter-intutive. Like if the turn is AGENT, then player make a move?

Reply:
Yeah definitely true. In another PR, I am thinking of changing all `agent` and `AGENT` keyword to `ally` and `ALLY` and leave `enemy` keyword as it is. How does that sound?